resources/configs/simple_flop_config.json

```diff
@@ -51,5 +51,7 @@
     "merging_threshold": 0.1,
     "rake_cap": 0.0,
     "rake_rate": 0.0
-  }
+  },
+  "added_lines": [],
```

Comment:
are these needed? IIRC these were optional in my serializing/deserializing. I'm happy to make them required